apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevRequest.kt

```diff
@@ -1,29 +1,28 @@
 package no.nav.etterlatte.brev.model
 
 import no.nav.etterlatte.brev.adresse.RegoppslagResponseDTO
-import no.nav.etterlatte.libs.common.brev.model.Adresse
 import no.nav.etterlatte.libs.common.soeknad.dataklasser.common.Spraak
 import java.time.LocalDate
 
 abstract class BrevRequest {
     abstract val spraak: Spraak
     abstract val avsender: Avsender
-    abstract val mottaker: Mottaker
+    abstract val mottaker: MottakerRequest
     abstract val attestant: Attestant?
     val utsendingsDato = LocalDate.now()
     abstract fun templateName(): String
 }
 
 // TODO: Sikre non-nullable
-data class Mottaker(
+data class MottakerRequest(
```

Comment:
Litt uheldig navn siden det blir litt "requestception". Kunne brukt importalias de få stedene det blir konflikt. 

Eventuelt får jeg ta en større opprydding i modellene ved en senere anledning.

Reply:
Sant, dette namnet er ikkje bra. Blir det riktig å kalle det `MottakerDTO`?

Reply:
Sånn som det er no ligg iallfall begge to i samme pakke, så dei kan ikkje få samme namn utan at vi også romsterer litt med pakkestrukturen.